python/sglang/srt/layers/moe/ep_moe/token_dispatcher.py

```diff
@@ -113,6 +119,28 @@ def set_dispatch_mode_as_low_latency(cls):
         cls._dispatch_mode = DeepEPDispatchMode.LOW_LATENCY
 
 
+class _DeepEPConfig:
+    _instance = None
+
+    def __init__(self):
+        config_str = global_server_args_dict["deepep_config"]
+        if config_str:
+            config_parsed = load_json_config(config_str)
+            if torch.distributed.get_rank() == 0:
+                logger.info(f"Use DeepEP Config: {config_parsed}")
+            self.normal_dispatch_config = Config(**config_parsed["normal_dispatch"])
+            self.normal_combine_config = Config(**config_parsed["normal_combine"])
+        else:
+            self.normal_dispatch_config = None
+            self.normal_combine_config = None
+
+    @classmethod
+    def get_instance(cls):
```

Comment:
how about rename it as `get_dispatch_config` and `get_combine_config`?

Reply:
I'm OK for both, though that will introduce two new functions, thus extra indirections and make code slightly more verbose.

Reply:
Resolved after discussion